### DISCUSSION TITLE
Power control greyed out on one inverter

### DISCUSSION THREAD

**Discussion by petehhhhh** [WillCodeForCats/solaredge-modbus-multi#447] [Q&A]
I have two inverters. The master has a battery and a meter connected. I can use the power control options on this via the integration. For the second inverter, the  option to limit control the inverter is always greyed out. If I go into installer menu on the inverter, I can set a site production limit for this inverter (so it is supported) but not via the HA integration?

**Comment by WillCodeForCats** [comment]
The integration supports whatever the inverter allows access to. Not all settings for all inverters are allowed over modbus.

**Comment by WillCodeForCats** [comment]
Upload your diagnostic file and a debug log showing the initialization and I can look.
